src/extensions/mvs/camera.ts

```diff
@@ -51,34 +52,54 @@ export async function setCamera(plugin: PluginContext, params: MolstarNodeParams
     await PluginCommands.Camera.SetSnapshot(plugin, { snapshot });
 }
 
+async function focusBoundingSphere(plugin: PluginContext, params: MolstarNodeParams<'focus'>, boundingSphere: Sphere3D | undefined, extraRadius: number) {
```

Comment:
What is the `extraRadius`?

Reply:
value added to the bounding sphere radius of the target.